Fibonacci.java

```diff
@@ -0,0 +1,38 @@
+import java.util.Scanner;
+
+public class Fibonacci {
+  public static void main(String[] args) { 
+    System.out.println("Enter an integer : ");
+    Scanner input = new Scanner(System.in);
+    int number = input.nextInt();
+    while (number < 0) {
+      /*if the number is negative
+        request to enter again*/ 
+      System.out.println("Wrong!!!(Your number is negative)Try again :");
```

Comment:
NO!!!!! NEVER BE EMOTIONAL WITH END USER!!!!!!!!!!!!!!! Never!!!!!!!!!!!!!!!! 

Doesn't look polite, right?